src/node.h

```diff
@@ -225,6 +225,10 @@ class Environment;
 class MultiIsolatePlatform : public v8::Platform {
  public:
   virtual ~MultiIsolatePlatform() { }
+  // Returns true if work was dispatched or executed. New tasks that are
+  // posted during flushing of the queue are postponed until the next
+  // flushing.
+  virtual bool FlushForegroundTasks(v8::Isolate* isolate) = 0;
```

Comment:
This is technically a breaking change, even though it only targets embedders. They are likely still compiling Node from source anyway at this point, so I think this is okay, but it's worth pointing out.